apps/modernization-ui/src/generated/graphql/schema.ts

```diff
@@ -2214,7 +2214,7 @@ export enum RecordStatus {
   Active = 'ACTIVE',
   Inactive = 'INACTIVE',
   LogDel = 'LOG_DEL',
-  Superceded = 'SUPERCEDED'
+  Superseded = 'SUPERSEDED'
 }
```

Comment:
Anything in the `/generated` folder is auto generated based on the Java API declaration. The `SUPERCEDED` value corresponds to what is stored in the database and shouldn't be changed.